build/images/ovs/README.md

```diff
@@ -6,13 +6,16 @@ of Antrea (such as IPSec) require a recent version of OVS, more recent than the
 version included in Ubuntu 18.04. The built image is then used as the base image
 for the Antrea main Docker image.
 
-## Building the image and pushing it to Dockerhub
+The image is re-built and pushed to Dockerhub periodically (every 6 hours) by a
+[Github workflow](/.github/workflows/update_ovs_image.yml).
+
+## Manually building the image and pushing it to Dockerhub
 
 Choose the version of OVS you want to build by setting the `OVS_VERSION`
 environment variable. Then run the `build_and_push.sh` script. For example:
 
 ```bash
-OVS_VERSION=2.11.1 ./ovs/build_and_push.sh
+OVS_VERSION=2.11.1 ./build_and_push.sh
```

Comment:
It does not seem implicit to me that we're in the build/images/ovs directory - for the "antrea deployer in a hurry" that just wants to cut and paste commands perhaps this might not be obvious

Reply:
I updated the instructions (note that these instructions are here only for the very unlikely case that a maintainer needs to push the image manually).